python/lsst/pipe/base/pipeline.py

```diff
@@ -501,6 +501,91 @@ def fromPipeline(cls, pipeline: Pipeline) -> Pipeline:
     def __str__(self) -> str:
         return str(self._pipelineIR)
 
+    def mergePipeline(self, pipeline: Pipeline) -> None:
+        """Merge another in-memory `Pipeline` object into this one.
+
+        This merges another pipeline into this object, as if it were declared
+        in the import block of the yaml definition of this pipeline. This
+        modifies this pipeline in place.
+
+        Parameters
+        ----------
+        pipeline : `Pipeline`
+            The `Pipeline` object that is to be merged into this object
+        """
+        self._pipelineIR.merge_pipelines((pipeline._pipelineIR,))
+
+    def addLabelToSubset(self, subset: str, label: str) -> None:
+        """Add a task label from the specified subset.
+
+        Parameters
+        ----------
+        subset : `str`
+            The labeled subset to modify
+        label : `str`
+            The task label to remove from the specified subset
+
+        Raises
+        ------
+        ValueError : Raised if the specified subset does not exist within the
+                     pipeline.
+                     Raised if the specified label does not exist within the
+                     pipeline.
+        """
+        if label not in self._pipelineIR.tasks:
+            raise ValueError(f"Label {label} does not appear within the pipeline")
+        if subset not in self._pipelineIR.labeled_subsets:
+            raise ValueError(f"Subset {label} does not appear within the pipeline")
+        self._pipelineIR.labeled_subsets[subset].subset.add(label)
+
+    def removeLabelFromSubset(self, subset: str, label: str) -> None:
+        """Remove a task label from the specified subset.
+
+        Parameters
+        ----------
+        subset : `str`
+            The labeled subset to modify
+        label : `str`
+            The task label to remove from the specified subset
+
+        Raises
+        ------
+        ValueError : Raised if the specified subset does not exist in the
+                     pipeline.
+                     Raised if the specified label does not exist within the
+                     specified subset.
+        """
+        if subset not in self._pipelineIR.labeled_subsets:
+            raise ValueError(f"Subset {label} does not appear within the pipeline")
+        if label not in self._pipelineIR.labeled_subsets[subset].subset:
+            raise ValueError(f"Label {label} does not appear within the pipeline")
+        self._pipelineIR.labeled_subsets[subset].subset.remove(label)
+
+    def findLabelInSubsets(self, label: str) -> set[str]:
```

Comment:
This method name doesn't sit quite right with me. I think the order of importance is switched; at a glance, it looks like I'm finding a label. I think something like `findSubsetsFromLabel` would work better.